invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/modules/dropdown.overrides

```diff
@@ -77,3 +77,17 @@
     }
   }
 }
+
+.mobile-table-cell .ui.dropdown{
+  @media all and (max-width: @largestMobileScreen) {
+    min-width: 0em !important;
+
+    & > .menu {
+      min-width: 14em !important;
+      left: auto !important;
+      right: 0px !important;
+      border-top-width: 1px !important;
```

Comment:
why are these needed? could you avoid using `important` ?

Reply:
The rule I'm overriding was marked as important, so I don't think I can. This rule is needed because in semantic ui react, as the dropdown should have the same width than the dropped text, not top margin is required.

Reply:
The problem with this approach is that there is also a blue line on the bottom of the dropdown when you expand (see screenshot 3). I'm not a fan of this, as it is not consistent with the other views, but after a while trying things I could only find one alternative solution (using before) and it feels quite hackish.

Reply:
where was it marked as important ?